server/configuration.go

```diff
@@ -85,8 +86,13 @@ func (p *Plugin) validateConfiguration(configuration *configuration) error {
 			return errors.New("failed to get the size of whitelist from the DB")
 		}
 
-		if configuration.ConnectedUsersAllowed < whitelistSize {
-			return errors.New("failed to save configuration, no. of connected users allowed should be greater than or equal to the current size of the whitelist")
+		invitedSize, err := p.store.GetSizeOfInvitedUsers()
+		if err != nil {
+			return errors.New("failed to get the size of invited users from the DB")
+		}
+
+		if configuration.ConnectedUsersAllowed < (whitelistSize + invitedSize) {
+			return errors.New("failed to save configuration, no. of connected users allowed should be greater than or equal to the current size of the whitelist and invited users")
 		}
```

Comment:
I'm wondering if we should just rip out this check altogether. It might be hard for the enduser to see what the current size of each is, and presumably if the `ConnectedUsersAllowed` is too small, it just means no one new can join? (Might simplify this code path!)

Reply:
I'm not opposed to that. I had added similar check to validate that the invite pool size didn't go over seats available, but that was a bit over the top and might have caused problems towards the top end of max connected users.

Similarly, I guess this check in question could cause the same issue... 

Let's say max 200, with 100 seats open, and 100 invites then get sent out... if non-invited users connect (as allowed because `whitelistSize >= a.p.getConfiguration().ConnectedUsersAllowed` is still false), that would cause this check to fail.